content/rdi/installation/install-k8s.md

```diff
@@ -14,10 +14,36 @@ Throughout the document, the snippets make use of the Kubernetes `kubectl` tool.
 
 ## Prerequisites
 
-- An existing Redis Enterprise cluster version >= 6.2.
-- [RedisGears](https://redis.com/modules/redis-gears/) >= 1.2.5 installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install.
+- An existing Redis Enterprise cluster version >= 6.2.18
+- [RedisGears](https://redis.com/modules/redis-gears/) {{<param rdi_redis_gears_current_version>}} installed on the cluster. In case it's missing, see [Install RedisGears for Redis Data Integration]({{<relref "/rdi/installation/install-redis-gears.md">}}) to install. 
 - A target Redis DB (can be added after installation).
 
+> Note: The Gears binaries to install must match the base OS of REC containers. In case of [Rancher](https://www.rancher.com/), REC base OS will be Ubuntu 18.04. Use the following command to install Gears
+
+```bash
+curl -s https://redismodules.s3.amazonaws.com/redisgears/redisgears.Linux-ubuntu18.04-x86_64.1.2.6-withdeps.zip -o /tmp/redis-gears.zip
+```
+
+In case the wrong Gears binaries had been installed, use the following commands to fix it
+
+```bash
+# start port forwarding to the rec api
+kubectl port-forward service/rec 9443:9443
+
+# find the uid of the gears module
+# note: skip piping to the jq if it is not installed
+curl -k -v -u "user:pwd" https://localhost:9443/v1/modules | jq '.[] | {module_name,uid,semantic_version}'
```

Comment:
jq might be missing, should be installed beforehand

Reply:
he has a comment there